### PR TITLE
Bump CI deps & Drop net461

### PR DESCRIPTION
**Pull request by Pliner**
:-/

### CODE REVIEW

File: Source/EasyNetQ.Management.Client.sln [hunk 1]
```diff
@@ -16,6 +16,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution items", "Solution
 		..\Tools\build\build.ps1 = ..\Tools\build\build.ps1
 		..\license.txt = ..\license.txt
 		..\README.md = ..\README.md
+		..\appveyor.yml = ..\appveyor.yml
+		..\appveyor.yml = ..\.travis.yml
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client.sln:20]: изображение

Comment (Pliner) [reply Source/EasyNetQ.Management.Client.sln:20]: )))))))

Comment (Pliner) [reply Source/EasyNetQ.Management.Client.sln]: Fixed.

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ language: csharp
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: bionic
       mono: latest
-      dotnet: 2.1.3
+      dotnet: 3.0
```
Comment (sungam3r) [inline .travis.yml:7]: 3.1 ?

Comment (Pliner) [reply .travis.yml:7]: It seems 3.1 is not supported yet, but I will check.

File: Source/EasyNetQ.Management.Client.IntegrationTests/TestExtensions.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@ namespace EasyNetQ.Management.Client.IntegrationTests
 {
     public static class TestExtensions
     {
-        public static T ShouldNotBeNull<T>(this T obj) where T:class
+        public static T ShouldNotBeNull<T>(this T obj) where T : class
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client.IntegrationTests/TestExtensions.cs:8]: https://github.com/shouldly/shouldly

Comment (Pliner) [reply Source/EasyNetQ.Management.Client.IntegrationTests/TestExtensions.cs:8]: Prefer FluentAssertions)))

**Pull request merged**